### DISCUSSION TITLE
too slow in nextjs

### DISCUSSION THREAD

**Discussion by keshavgautam** [vercel/next.js#13269] [Help]
https://ecomapp-tfacupa7sq-uc.a.run.app/ 
this the website and its too slow

**Comment by leerob** [comment]
Could you provide a specific question – are you wondering how to improve the performance of your Next.js application? 

Consider utilizing [pre-rendering](https://nextjs.org/docs/basic-features/pages#pre-rendering) to server static pages. These will have the best performance placed behind a CDN like [Vercel](https://vercel.com).
